### PR TITLE
fix: use latest multicodec release

### PR DESCRIPTION
**Pull request by vmx**
Using internal modules is a bad idea, but this is a quick fix in order
to not break the world.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -3,7 +3,8 @@
 const mh = require('multihashes')
 const multibase = require('multibase')
 const multicodec = require('multicodec')
-const codecs = require('multicodec/src/base-table.json')
+const { baseTable } = require('multicodec/src/base-table.js')
```
Comment (Gozala) [inline src/index.js:6]:
```suggestion
const { baseTable: codecs } = require('multicodec/src/base-table.js')
```

File: src/index.js [hunk 2]
```diff
@@ -3,7 +3,8 @@
...
+const codecs = baseTable
```
Comment (Gozala) [inline src/index.js:7]:
```suggestion
```

**Review by Gozala — APPROVED** [review]
Looks good to me

File: src/index.js (hunk 1 above)
Comment (vmx) [reply src/index.js:6]: Oh sweet, I was hoping there would be syntax for that. Nice!

**Review by hacdias — APPROVED** [review]
I will work on a quick PR to export different kinds of tables and if everyone agrees, we can expose them safely.

**Comment by vmx** [comment]
@hacdias I released with this one though, so that @Gozala can work off a released version. We can work on using proper exports next week.

**Pull request merged**